Dockerfile

```
@@ -26,14 +26,11 @@ COPY lib lib
 COPY assets assets
 COPY priv priv
 
-# grab images
 COPY Makefile Makefile
 RUN make content
 
 RUN npm run --prefix ./assets deploy
-RUN mix phx.digest
-
-RUN mix do compile, release
+RUN mix do phx.digest, compile, school_house.gen.sitemap, school_house.gen.rss, release
```

Thoughts on breaking up this line?

```suggestion
RUN mix do phx.digest, \
    compile, \
    school_house.gen.sitemap,  \
    school_house.gen.rss, \
    release
```